db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb

```diff
@@ -0,0 +1,23 @@
+class MosquitoGenomeUpdate20191002 < ActiveRecord::Migration[5.1]
+  def up
+    # The previous mosquito genome was not added via a migration so we need to destroy it
+    old_hg = HostGenome.find_by(name: "Mosquito").destroy
```

Comment:
Won't this break references between all current samples and their host genomes? (foreign key constraints should prevent this) 

Why not create a Mosquito v2?